python/tarfile-extractall-traversal.py

```diff
@@ -32,6 +32,9 @@ def extract_3(args):
     tf = tarfile.open(path)
     tf.extractall(save_path)
 
+def extract_4(args, tarobj):
+    tf = tarfile.open(mode='r', fileobj=None)
```

Comment:
Looks like you are missing `# ruleid: tarfile-extractall-traversal` to tell semrep that line #36 should be reported as a finding